frontend/packages/helm-plugin/src/actions/types.ts

```diff
@@ -1,6 +1,9 @@
+import { HelmRelease } from '../types/helm-types';
+
+type HelmActionObj = { name: string; namespace: string; version?: number | string };
```

Comment:
`version` is not optional in the `HelmRelease` why is it optional here? won't a helm release always contain a version?